platform/viewer/src/components/UserPreferences/GeneralPreferences.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useSelector } from 'react';
+import PropTypes from 'prop-types';
+
+import i18n from '@ohif/i18n';
+
+import { TabFooter, LanguageSwitcher, useSnackbarContext } from '@ohif/ui';
+import { useTranslation } from 'react-i18next';
+
+import './GeneralPreferences.styl';
+
+const languages = [
+  // TODO: list of available languages should come from i18n.options.resources
```

Comment:
We have `i18n` here. Any reason we can't satisfy this todo stub?

Reply:
So, the i18n have the list of language's codes but it does not have a map between language and its description e.g. `pt-BR <-> português`

Reply:
Can we create that map for a few that we intend to support?

forEach i18n, use label as mapped friendly description, use value as i18n value?

Reply:
Code changes done. Now we have an exposed availableLanguages with naming on i18n that we can use.